apps/test-suite/tests/NewNotifications.js

```diff
@@ -918,6 +918,44 @@ export async function test(t) {
         10000
       );
 
+      t.it(
+        'schedules a repeating daily notification. only first scheduled event is verified.',
+        async () => {
+          const dateNow = new Date();
+          const trigger = {
+            hour: dateNow.getHours(),
+            minute: dateNow.getMinutes() + 2,
+          };
+          await Notifications.scheduleNotificationAsync({
+            identifier,
+            content: notification,
+            trigger,
+          });
+          const result = await Notifications.getAllScheduledNotificationsAsync();
+          if (Platform.OS === 'android') {
+            t.expect(result[0].trigger).toEqual({
+              type: 'daily',
+              ...trigger,
+            });
+          } else if (Platform.OS === 'ios') {
+            t.expect(result[0].trigger).toEqual({
+              type: 'calendar',
+              class: 'UNCalendarNotificationTrigger',
+              repeats: true,
+              dateComponents: {
+                ...trigger,
+                timeZone: null,
+                isLeapMonth: false,
+                calendar: null,
+              },
+            });
```

Comment:
Since we use the `UNCalendarNotificationTrigger` on iOS, the validation between Android and iOS differs but that is expected so while not ideal, should needs to be this way.